src/util/report.py

```diff
@@ -286,6 +286,21 @@ def ref(xid):
     except Exception:
         _logger.warning("Cannot announce message", exc_info=True)
 
+    # Chat window with the report will be open post-upgrade for the admin
+    if version_gte("9.0"):
+        if version_gte("saas~16.3"):
+            channel_member_model = "discuss.channel.member"
+        elif version_gte("16.0"):
+            channel_member_model = "mail.channel.member"
+        else:
+            channel_member_model = "mail.channel.partner"
+        try:
+            registry[channel_member_model].search(
+                [("partner_id", "=", user.partner_id.id), ("channel_id", "=", recipient.id)]
+            )[0].with_context(ctx).fold_state = "open"
+        except Exception:
+            _logger.warning("Cannot open chat window", exc_info=True)
```

Comment:
I don't think we need to log anything, in any case if we do logging exception info is better done with `_logger.exception` ;)
```suggestion
    pass
```
Maybe use `contextlib.suppress` and be more specific: what kind of exception we get here? and why?